src/components/Homepage/Homepage.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from "react";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import Button from "react-bootstrap/Button";
+import "./homepage.css";
+import CanvasJSReact from "../../assets/canvasjs.react";
+
+const CanvasJS = CanvasJSReact.CanvasJS;
+const CanvasJSChart = CanvasJSReact.CanvasJSChart;
+
+interface Props {}
+
+interface State {}
+
+export default class Homepage extends Component<Props, State> {
+  state = {};
+
+  render() {
+    const options = {
+      exportEnabled: true,
+      animationEnabled: true,
+      title: {
+        text: "Budget - March 2020: €2,235.00",
+        fontFamily: "Merriweather"
+      },
+
+      data: [
+        {
+          type: "pie",
+          // showInLegend: "true",
+          legendText: "{label}",
+          indexLabelFontSize: 16,
+          indexLabel: "{label} - #percent%",
+          percentFormatString: "#0.##",
+          yValueFormatString: "€#,###.00",
+          indexLabelFontFamily: "Caladea",
+          toolTipContent: "{y} (#percent%)",
+          dataPoints: [
+            { y: 670.5, label: "Savings Target" },
+            { y: 340.84, label: "Transportation" },
+            { y: 657.09, label: "Rent" },
+            { y: 234.68, label: "Groceries" },
+            { y: 93.87, label: "Clothing" },
+            { y: 111.75, label: "Donations" },
+            { y: 46.94, label: "Misc." },
+            { y: 79.34, label: "Remaining Budget" }
+          ]
+        }
+      ]
+    };
```

Comment:
get all this out to a `chartConfigs.js` file which exports this object

Reply:
Again: no use of state + no use of lifecycle methods => functional component

Reply:
Will do!